docs/src/components/home/sections/PrimitiveSection.tsx

```diff
@@ -0,0 +1,245 @@
+              name="language"
+              value={radioValue}
+              direction="row"
+              gap="small"
+              onChange={(e) => setRadioValue(e.target.value)}
+              labelHidden
+            >
+              <Radio value="html">html</Radio>
+              <Radio value="css">css</Radio>
+              <Radio value="javascript">javascript</Radio>
+            </RadioGroupField>
+          </Center>
+        </Card>
+        <Card variation="elevated" flex="1">
+          <Center>
+            <SliderField
+              label="Slider"
+              labelHidden
+              min={0}
+              max={100}
+              step={1}
+              defaultValue={50}
+            />
+          </Center>
+        </Card>
+        <Card variation="elevated" flex="1">
+          <Center>
+            <CheckboxField
+              label="Sprinkles"
+              value="Sprinkles"
+              name="topping"
+              defaultChecked
+            />
+          </Center>
+        </Card>
+      </Flex>
+    </Flex>
+  );
+};
+
+export const ComingSoonPrimitiveSection = ({ platform }) => {
+  return (
+    <View as="section" className="docs-home-section docs-grid-bg">
+      <Flex direction="column" className="docs-home-subsection--thin">
+        <Heading level={2} textAlign="center">
+          {upperFirst(platform)} primitive components{' '}
+          <strong>coming soon</strong>!
+        </Heading>
+        <View className="docs-home-coming-soon">
+          <Preview />
+        </View>
+      </Flex>
+    </View>
+  );
+};
+
+export const PrimitiveSection = ({ platform, ...rest }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const entry = useIntersectionObserver(ref, {
+    threshold: 0.125,
+    freezeOnceVisible: true,
+  });
+  const isVisible = !!entry?.isIntersecting;
+
+  return (
+    <View
+      as="section"
+      className={classNames(
+        'docs-home-section',
+        'docs-grid-bg',
+        'fade-in',
+        isVisible && 'shown'
+      )}
+      backgroundColor="background.secondary"
+      ref={ref}
+    >
+      <Flex direction="column" gap="large" className="docs-home-subsection">
+        <Heading level={2}>
+          Speed up development with over <br />
+          <strong>40 production-ready components</strong>
+        </Heading>
+        <Flex
+          direction={{
+            base: 'column',
+            xl: 'row',
+          }}
+          gap="xl"
+          alignItems="stretch"
+        >
+          <Flex direction="column" flex="1">
+            <Text className="docs-home-text">
+              Amplify UI components are built with plain React and CSS to
+              provide a solid foundation for building UIs and design systems.
+              These components are theme-able, composable, reusable. They play
+              nicely with other UI components or styling frameworks. Interactive
+              components can be controlled and uncontrolled.
```

Comment:
Minor revisions (suggested): "These components are themeable, composable, reusable, and play nicely with other UI components or styling frameworks. ~~Interactive components can be controlled and uncontrolled.~~"